tests/features/eap.feature

```diff
@@ -0,0 +1,9 @@
+@jboss-eap-7/eap70-openshift
```

Comment:
Should this apply to all images?

Reply:
Actually yes. I could move it to cct_module repo.